service/routetable/route_table.go

```diff
@@ -71,6 +71,10 @@ func (r *RouteTable) IdForName(ctx context.Context, name string) (string, error)
 	return id, nil
 }
 
+func (r *RouteTable) Names() []string {
+	return r.names
+}
```

Comment:
Just adding a getter to make the configuration available. That helps with the adapters.

Reply:
I missed that in the previous PR but having `names` property in this service makes little sense to me. This service allows to get route table ID for arbitrary name. It will memorize the ID for an arbitrary name.

Reply:
The route table service implementation is injected into the adapters. There we iterate over all route tables we have. We could inject the names and the service separately but the service is already configured with them. When we dropped `Boot` it made less sense because we do not iterate over the names in the initial lookup anymore. I am not big on changing this now again though.

Reply:
I'd like the names to be injected separately. If you want to change that in the future this is fine with me.

Reply:
Alright. I do it later.